packit/upstream.py

```diff
@@ -297,12 +297,14 @@ def get_current_version(self) -> str:
                 "There are no tags in the repo, `git describe` will very likely fail."
             )
 
-        ver = self.command_handler.run_command(
+        # raw_version - version before processing by Upstream.get_version_from_tag()
+        raw_ver = self.command_handler.run_command(
```

Comment:
Not sure if the comment gives us any new value.
```suggestion
        raw_ver = self.command_handler.run_command(
```

Reply:
fixed